dashboard/src/components/PackageLocationCard.vue

```diff
@@ -30,31 +30,31 @@ const choose = async () => {
       <div v-if="failed" class="alert alert-danger" role="alert">
         Move operation failed, try again!
       </div>
-      <div v-if="packageStore.isMoving" class="alert alert-info" role="alert">
+      <div v-if="ingestStore.isMoving" class="alert alert-info" role="alert">
         The package is being moved into a new location.
       </div>
       <h4 class="card-title">Location</h4>
       <p class="card-text">
-        <span v-if="packageStore.isRejected">Package rejected.</span>
-        <span v-else-if="!packageStore.current?.locationId"
+        <span v-if="ingestStore.isRejected">Package rejected.</span>
```

Comment:
All of these methods (`isMoving()`, `isRejected()`, `isMovable()`) sound odd in the context of the "ingestStore" because "ingest" isn't a noun.  I think using "sipStore" would make a little more sense in a lot of these cases (e.g. `sipStore.isRejected()`) and maybe we could avoid the changing `packageStore.current` to `ingestStore.currrentSip` and similar.  Of course `sip.isRejected()` would read even better...

Reply:
Good point. Because we have the storage store already, I thought about having a general ingest store, in case we add other entities in the future, and I was planning to put both (AIPs and locations) into the storage store.

Now that you say this, I think it makes more sense to have a store per entity (`sips`, `aips` and `locations` stores). Not sure if we should use plurals here too now that they will be dealing with entities/collections, but let me know your thoughts, I'll do this in a following PR when we add AIPs.

Reply:
Having a store per entity sounds good to me.  I like the singular version of the entity names for code libraries (e.g. `sip`, `aip`, `location`) but I'd be okay with plurals too.